src/collections/FormSubmissions/hooks/sendEmail.ts

```diff
@@ -82,9 +84,19 @@ const sendEmail = async (beforeChangeData: any, formConfig: PluginConfig) => {
               try {
                 const emailPromise = await payload.sendEmail(email);
                 return emailPromise;
-              } catch (err) {
+              } catch (err: any) {
                 console.error(`Error while sending email to address: ${to}. Email not sent.`);
-                console.error(err);
+                if (
+                  err.response &&
+                  err.response.body &&
+                  err.response.body.errors
+                ) {
+                  err.response.body.errors.forEach((error: any) =>
```

Comment:
Instead of looping here just take the first index, `err.response.body.errors?.[0]?.message`